src/utils.rs

```diff
@@ -181,7 +206,7 @@ pub fn parse_args() -> AppConfig {
         .arg(
             Arg::with_name("fill")
             .long("fill")
-            .conflicts_with_all(&["horizontal_align", "vertical_align", "margin"])
+            .conflicts_with_all(&["horizontal_align", "vertical_align", "margin", "offset"])
```

Comment:
Technically we could have an offset even with `fill`. Opinions?

Reply:
I thought about this initally, but I think this would have to be a separate option. The offset just moves the starting draw position. We would have to shrink the fill box to make this work. That would be simple enough for the sides, since most (but not all) WMs have the same decorations on the sides. But what about the top and bottom? Do we have a titlebar? Is the bottom the same as the sides or does it have extra decorations (like handle, etc)? We would need pass values for all four sides to make this work, I think.

Reply:
That's a good point. Ok, let's leave it like this for now.